curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/655b4dad1d38ff7cdd65cbfe.md

```diff
@@ -1462,6 +1464,7 @@ const playPreviousSong = () => {
 };
 
 const shuffle = () => {
+  
```

Comment:
the solution also need to have the same change of adding ` if (userData.songs.length === 0) return;`